client/lib/performance-tracking/with-stop-performance-tracking-prop.jsx

```diff
@@ -11,10 +11,10 @@ import { getSectionName } from 'state/ui/selectors';
 
 export const withStopPerformanceTrackingProp = ( () => {
 	return connect( null, {
-		stopPerformanceTracking: () => ( dispatch, getState ) => {
+		stopPerformanceTracking: ( metadata ) => ( dispatch, getState ) => {
```

Comment:
It may be beneficial to make the `metadata` parameter optional.

```suggestion
		stopPerformanceTracking: ( metadata = {} ) => ( dispatch, getState ) => {
```